jest.config.js

```diff
@@ -1,3 +1,10 @@
-module.exports = {
+export default {
   setupFilesAfterEnv: ['./jest.setup.js'],
+  moduleFileExtensions: ['js', 'ts', 'mjs'],
+  transform: {
+    '^.+\\.[t|j]sx?$': 'babel-jest',
+  },
+  moduleNameMapper: {
+    '#(.*)': '<rootDir>/node_modules/$1',
```

Comment:
inspired by https://github.com/chalk/chalk/issues/532

Reply:
We may want to consider moving from jest to vitest, as the latter has much better support for ESM. I've been converting all my OSS repos from jest to vitest, and the conversion is very easy.